webpack.config.js

```diff
@@ -19,6 +20,16 @@ const config = {
         }
       }
     ]
-  }
+  },
+  externals: [
```

Comment:
Is it necessary to import `isomorphic-fetch` in node in parallel to this lib? If that's the case, could you maybe also add a description about this to the readme?

Reply:
@AlexBHarley